### PR TITLE
Minimize rerender from gasprice

### PR DESCRIPTION
**Pull request by HenryNguyen5**
This prevents the entire tab from re-rendering on gas price slider changes via removing passed down state props and instead directly connecting the components using the prop.

### CODE REVIEW

File: common/components/Header/components/GasPriceDropdown.tsx [hunk 1]
```diff
@@ -5,25 +5,32 @@ import DropdownShell from 'components/ui/DropdownShell';
...
   }
 
   public render() {
-    const { value } = this.props;
     return (
       <DropdownShell
         color="white"
         size="smr"
-        ariaLabel={`adjust gas price. current price is ${value} gwei`}
+        ariaLabel={`adjust gas price. current price is ${this.props.gasPrice.raw} gwei`}
```
Comment (dternyak) [inline common/components/Header/components/GasPriceDropdown.tsx]: can we keep the de-structure?

Comment (HenryNguyen5) [reply common/components/Header/components/GasPriceDropdown.tsx]: I can revert, but this prevents creating another variable that's only going to be used once anyway.

Comment (dternyak) [reply common/components/Header/components/GasPriceDropdown.tsx]: It get's used quite a bit in that component afaict.

Comment (dternyak) [reply common/components/Header/components/GasPriceDropdown.tsx]: Nevermind, across multiple methods. No real benefit.

**Pull request merged**